app/pages/projects/project-filtering-interface.jsx

```diff
@@ -68,10 +71,11 @@ class ProjectFilteringInterface extends Component {
             const meta = projects[0].getMeta();
             pages = meta.page_count;
             projectCount = meta.count;
+            pageSize = meta.page_size;
           }
-          this.setState({ projects, pages, projectCount });
+          this.setState({ projects, pages, projectCount, pageSize });
         } else {
-          this.setState({ projects: [], pages: 0, projectCount: 0 });
+          this.setState({ projects: [], pages: 0, projectCount: 0, pageSize: 20 });
```

Comment:
Yeah, it's not needed for the maths. This is the case where no projects are fetched, so I still don't get why having a default, sorry.